### DISCUSSION TITLE
How to set the executing image on a per job basis

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#8547] [Q&A]
Hey, I was wondering if there is a way to set the executing image on a per job basis. Even better if this available at job execution time

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1640520864119000?thread_ts=1640520864.119000&cid=C01U954MEER).*

**Comment by yuhan** [comment]
Assuming you're using Kubernetes we do have support for this - you can apply a tag to the job and it will override the image to use:
```python
@job(
  tags = {
    'dagster-k8s/config': {
      'container_config': {
        'image': 'foobar:latest'
      },
    },
  },
)
```

**Reply by aivanni** [reply]
Is it possible to set image at job execution time through config?
